src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_References/Elements_of_Arrays.java

```diff
@@ -0,0 +1,35 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_References;
+
+public class Elements_of_Arrays {
+    public static int numbersMultiplication(int[] array) {      //banana is the method; int[] a{3,8,9} - array of parameters
+        int product = 1; // initializing the first value of the product
+        int i = 0;   // initialization of i with 0
+        while (i < array.length) {  // while i { from 0....to length of a}
+            product = product * array[i];  // multiplying each element of the array with the next one
```

Comment:
It's hard to read code with all these comments, a good way is to write a coment before method and put all details about this method there

Reply:
These comments are written for a better personal understanding, they are just notes for me to understand what I did in the program.